### PR TITLE
made basic changes

### LINKED ISSUE #38: navbar updates (homepage)
1. make logo/brand name font size bigger
2. space out the navbar evenly 

Image

### PR DESCRIPTION
**Pull request by Rahnken**
increased Logo size, redefined with ID
fixed some minor inconsistencies

may need to revisit with an image instead of html markup for logo

some minor changes in css probably needed to improve readability and good practices

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -35,14 +35,13 @@
   <nav class="navbar" id="divNav">
     <div class="flex-nav">
       <div class="nav-header">
-        <h2><span>Way To</span> Health</h2>
+        <h2 id="nav-logo"><span>Way To </span>Health</h2>
```
Comment (Rahnken) [inline index.html:38]: Makes it easier to target specifically this element for the use of styling of the logo and not affecting other H2s on the page should they be used

File: index.html [hunk 2]
```diff
@@ -53,13 +52,6 @@ <h2><span>Way To</span> Health</h2>
             <a href="/pages/newsletter.html">Newsletter</a>
           </li>
           <li class="nav-link"><a href="/pages/gallery.html">Gallery</a></li>
-
```
Comment (Rahnken) [inline index.html:56]: Removed old / outdated code

File: styles/styles.css [hunk 3]
```diff
@@ -27,7 +27,12 @@
   padding: 0px 10px;
 }
 
-.nav-header span {
+#nav-logo {
```
Comment (Rahnken) [inline styles/styles.css:30]: target new id value and modify with rem to improve accessibility

File: styles/styles.css [hunk 4]
```diff
@@ -45,14 +50,15 @@
...
+  display: flex;
   height: 100vh;
   overflow: hidden;
+  justify-content: space-evenly;
 }
 
 .nav-link {
   list-style: none;
-  margin: 40px 0;
+  margin: 1.25rem 0;
```
Comment (Rahnken) [inline styles/styles.css:61]: adjust margin size to better align with nav bar sizing

**Review by Rahnken — COMMENTED** [review]
There are several 'changes' that are only prettier going through and updating the ' with " in the css files, I can adjust my settings if this is a problem in future

File: styles/styles.css [hunk 5]
```diff
@@ -97,13 +103,13 @@
...
+  /* <<<<<<< 40-update-carousel-arrowshomepage
   background-color: var(--bg-color);
   background-image: url('../assets/hero_boy-semi-final-2.png'); */
-/* ======= */
-  background-color: rgba(46, 45, 51, .6);
+  /* ======= */
+  background-color: rgba(46, 45, 51, 0.6);
   background-image: url("../assets/hero_boy-semi-final-2.png");
-/* >>>>>>> main */
+  /* >>>>>>> main */
```
Comment (Rahnken) [inline styles/styles.css:112]: I think this  may need to be addressed, it looks like it was a merge conflict that wasn't dealt with quite right but its not a big deal at this moment

**Comment by Iconians** [comment]
sorry it took longer to close this pr was double and triple checking things with your comments

**Pull request merged**